### DISCUSSION TITLE
Include data from database

### DISCUSSION THREAD

**Discussion by svenb78** [ThinkR-open/golem#703] [Q&A]
Hi all, 

I have a Shiny app made with golem, and so far I use the /data folder for the apps internal data. Now, I would like to transfer my data to an external database (Postgres) and import the data from there each time the app starts. My question: What is a good practice to do so? 

Thanks for help!

**Comment by ColinFay** [comment]
Hey, 

One of the way to do that is to connect to the db inside your app server function. 

See for example: https://github.com/ColinFay/golemexamples/blob/master/golemqlite/R/app_server.R#L7 or https://github.com/ColinFay/hexmake/blob/master/R/app_server.R#L13
